Formula/n/nexttrace.rb

```diff
@@ -22,8 +22,7 @@ class Nexttrace < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:  "be00c4943af521d667ba4dc21000c6f5aeb434ad4c30683e0fc0046549362f23"
   end
 
-  # use "go" again after https://github.com/nxtrace/NTrace-core/issues/247 is fixed and released
```

Comment:
this issue did not really get fixed, upstream just chose a `-checklinkname=0` workaround